### PR TITLE
feat: generic activity banners

### PR DESCRIPTION
**Pull request by rileymiller**

### CODE REVIEW

File: components/activities/activity-banner.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  return (
+    <div
+      className={`c-activity-banner ${
+        classKebab({
+          isClosed: !isBannerOpen,
+        })
+      }`}
+    >
+      {/* This was necessary to make the TS compiler happy, for some reason it wasn't liking the JSX format <Component activity={activityAlert.activity} />*/}
```
Comment (AMFTech512) [inline components/activities/activity-banner.tsx:104]: Maybe "Component" is a reserved JSX keyword or something?

File: components/activities/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import { gql, useContext, usePollingQuery } from "../../deps.ts";
+import { ActivityBannerContext } from "./context.ts";
+import { ActivityConnection } from "../../types/mod.ts";
+
+const ACTIVITY_CONNECTION_QUERY = gql`
+query AlertsReadyByType($status: String, $type: String)
+    {
```
Comment (AMFTech512) [inline components/activities/hooks.ts:7]: Why are the indents so big lol?

File: components/activities/poll-banner/poll-banner.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import stylesheet from "./poll-banner.scss.js";
+
+const PollBanner = (
+  { activity }: ActivityBannerProps<Poll>,
+) => {
+  useStyleSheet(stylesheet);
+  return (
+    <>
+      {
+        isPrediction(activity) ? <PredictionBanner poll={activity} /> : <BasePollBanner poll={activity} />
```
Comment (austinhallock) [inline components/activities/poll-banner/poll-banner.tsx:17]: Nitpick but does this need to be in a fragment? Or can you return line 17 directly

Comment (rileymiller) [reply components/activities/poll-banner/poll-banner.tsx:17]: doesn't need to be a fragment, I can rm

**Pull request merged**